sdk/storage/azure-storage-common/inc/azure/storage/common/access_conditions.hpp

```diff
@@ -58,4 +58,22 @@ namespace Azure { namespace Storage {
     Azure::Core::Nullable<std::string> LeaseId;
   };
 
+  /**
+   * @brief Specifies HTTP options for conditional requests based on ContentHash.
+   */
+  struct ContentHashAccessConditions
+  {
+    /**
+     * @brief Specify this header to perform the operation only if the resource's ContentHash
+     * matches the value specified.
+     */
+    Azure::Core::Nullable<ContentHash> IfMatch;
```

Comment:
`IfMatch` and `IfNoneMatch` is too confusing

Reply:
Any proposals?
How about `IfHashMatch` and `IfNoneHashMatch`?

Reply:
`IfMatchMatch` and `IfHashNoneMatch`?

or

`IfContentHashMatch` and `IfContentHashNoneMatch`?

Reply:
`IfContentHashMatch` and `IfContentHashNoneMatch` are better. Will make those changes.

Reply:
@JeffreyRichter @tg-msft Please raise your concern if any.